index.js

```diff
@@ -91,15 +84,16 @@ export default function pThrottle({limit, interval, strict, onDelay}) {
 			});
 		};
 
-		throttled.abort = () => {
+		signal?.throwIfAborted();
+		signal?.addEventListener('abort', () => {
```

Comment:
You need to remove this listener when the function is done, so not to create event listener leaks.

Reply:
It seem to have no way to know that the function is done?